redis/asyncio/connection.py

```diff
@@ -493,6 +493,7 @@ async def read_response(
                 and self.protocol in ["3", 3]
                 and not HIREDIS_AVAILABLE
             ):
+                read_timeout = None
```

Comment:
why do you need this?

Reply:
ah, debugging code which accidentally crept in, good catch!